changelogs/fragments/7401.yml

```diff
@@ -0,0 +1,2 @@
+fix:
+- Breadcrumb issue in the new application. ([#7401](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7401))
```

Comment:
What was the issue? This message needs to be a short sentence explaining what the fix does

Reply:
Could you may be update the descriptions to include a bit details?

Reply:
Update in the CHANGELOG may lead to dismiss of the approval. Do you think it is good enough to add more details in the description?